src/core/nodeOps.ts

```diff
@@ -74,49 +75,46 @@ export const nodeOps: RendererOptions<TresObject, TresObject> = {
       else if (instance.isBufferGeometry) instance.attach = 'geometry'
     }
 
+    instance.__tres = {
+      ...instance.__tres,
+      type: name,
+      memoizedProps: props,
+      eventCount: 0,
```

Comment:
**question:** What are `memoizedProps` and `eventCount` used for? I can't find them being used anywhere.

Reply:
I added `memoizedProps` to be used on disposal and HMR cases where keeping the original `args` prop is necessary to re-construct the instance. Is not used here but its needed on #466 and maybe #514 

I added `eventCount` to be used for #529